admin/uploads.rb

```diff
@@ -88,25 +91,45 @@
 	  end
     column "" do |upload|
       result = ""
-      result += link_to(t(:view), admin_upload_path(upload), class: "member_link edit_link view", title: I18n.t("active_admin.uploads.title1"))
-      result += link_to(t(:edit), edit_admin_upload_path(upload), class: "member_link edit_link edit", title: I18n.t("active_admin.uploads.title2"))
-      result += link_to(t(:delete), admin_upload_path(upload), method: :DELETE, "data-confirm" => t("delete_article", scope: [:goldencobra, :flash_notice]), class: "member_link delete_link delete", title: I18n.t("active_admin.uploads.title3"))
+      result += link_to(
```

Comment:
Your changes to incorporate our style guide are well received. Next time please put them into their own commit. It makes finding the actual change of the PR quite difficult.